### ISSUE TITLE
Do not swallow error message when prompt is enabled

### ISSUE THREAD

**Issue by IgnusG** [cenk1cenk2/listr2#2]
The error message is replaced with ['Cancelled the prompt'](https://github.com/cenk1cenk2/listr2/blob/b517867ec6ff7df6e3a757b932cc139fc35caf7c/src/lib/task.ts#L196-L199) when a task is to be marked as failed by the developer but a prompt was used in it. 

![image](https://user-images.githubusercontent.com/6438760/77766084-7962f380-703f-11ea-912f-aef24b01e22f.png)

Maybe there could be a `task.fail` method instead to distinguish between exceptions thrown from internal packages (if that is the concern) and custom exceptions (signalling a task has failed because of whatever the user put in).

**Comment by cenk1cenk2** [comment]
I can add the fail message as an output if desired result is that. Additional to the prompt fail.

**Comment by cenk1cenk2** [comment]
It does not swallow error messages in the prompt anymore. This was on my task list, glad it is sorted.

Output after successfully processing the prompt:
![image](https://user-images.githubusercontent.com/26881592/77795748-5d2a7b00-706e-11ea-80a0-03b235e1fac5.png)

```typescript
{
          task: async (ctx, task): Promise<any> =>{
            ctx.secondInput = await task.prompt<string>('Select',
              { message: 'Select some', choices: [ 'me', 'or me' ] }
            )
            throw new Error('I got the input but failed afterwards because of an unknown reason.')
          }
}
```

**Comment by cenk1cenk2** [comment]
Fixed with [v1.3.5](https://github.com/cenk1cenk2/listr2/releases/tag/v1.3.5), need more testing I did not have the environment for testing at the moment.

**Comment by IgnusG** [comment]
I'll try it out. Thanks a lot @cenk1cenk2! ❤️
